wp-content/civi-extensions/goonjcustom/Civi/InstitutionService.php

```diff
@@ -139,35 +139,48 @@ private static function addContactToGroup($contactId, $groupId) {
    * @param object $objectRef
    *   The parameters that were sent into the calling function.
    */
-  public static function setOfficeDetails($op, $groupID, $entityID, &$params) {
+  public static function organizationCreated(string $op, string $objectName, int $objectId, &$objectRef) {
 
-    if ($op !== 'create' || self::getOrgSubtypeName($entityID) !== self::ENTITY_SUBTYPE_NAME) {
-      return;
+    if ($op !== 'create' || $objectName !== 'Organization') {
+      return FALSE;
     }
 
-    if (!($stateField = self::findStateField($params))) {
-      return;
+    \Civi::log()->info('Organization created: ', [
+      'id' => $objectId,
+      'subtypes' => $objectRef->contact_sub_type,
+    ]);
```

Comment:
Why do we need logs here is there any case where this code breaks and you want these logs? I don't think we needed if you want try to use try catch blocks.

Why are you not removing logs, before giving them for review please remove them